code/modules/surgery/organs/autosurgeon.dm

```diff
@@ -103,9 +103,55 @@
 	uses = 1
 	starting_organ = /obj/item/organ/cyberimp/arm/syndie_mantis
 
-/obj/item/autosurgeon/organ/syndicate/syndie_mantis/l
+/obj/item/autosurgeon/organ/syndicate/syndie_mantis/AltClick(mob/living/user) //Basically this combines screwdriver with an insert_organ proc because I can't be assed
```

Comment:
why not make it a generic behavior or something locked behind a variable on the base type? doing it this way just makes it so any changes need to be copypasted across all the times you make the same thing

Reply:
also this ability should be listed as an addendum in examine()